casbin/rbac/default_role_manager/role_manager.py

```diff
@@ -47,9 +47,11 @@ def clear(self):
 
     def add_link(self, name1, name2, *domain):
         if len(domain) == 1:
-            if len(domain[0]) > 1:
+            if len(domain[0]) > 0:
```

Comment:
Why change like this? Why just revert this part to the original code?

![image](https://user-images.githubusercontent.com/3787410/114115008-94d03200-9914-11eb-9784-56c1c5c35b3f.png)